lib/ar_after_transaction/version.rb

```diff
@@ -1,3 +1,3 @@
 module ARAfterTransaction
```

Comment:
should add frozen string comment here ?

Reply:
Yeah, I can create new PR with small fixes